### PR TITLE
Properly describe the thing being inserted or selected by SocketDetails popup

### PR DESCRIPTION
**Pull request by robojumper**
After changing it for shaders and ornaments, I realized that `SocketDetailsSelectedPlug` is responsible for quite a few more kinds of items. This adds the rest of the names for kinds of plugs.

### CODE REVIEW

**Comment by robojumper** [comment]
This is the first time I'm interacting with the `i18next-scanner`, so I'm not sure if this is the preferred approach, cc @delphiactual.
Also not too sure about the "Insert/Apply/Equip" wording -- I used "Apply" where there's always only one of those slots on the item, "Insert" for multiple equivalent sockets, and "Equip" for the ability sockets.

File: config/i18n.json [hunk 1]
```diff
@@ -916,12 +916,22 @@
...
+    "InsertAbility": "Equip Ability",
+    "SelectAbility": "Select Ability",
+    "InsertFragment": "Insert Fragment",
+    "SelectFragment": "Select Fragment",
+    "InsertAspect": "Insert Aspect",
+    "SelectAspect": "Select Aspect",
+    "InsertTransmat": "Apply Transmat Effect",
+    "SelectTransmat": "Select Transmat Effect",
+    "InsertProjection": "Apply Ghost Projection",
+    "SelectProjection": "Select Ghost Projection",
```
Comment (delphiactual) [inline config/i18n.json:934]:
```suggestion
    "Insert":  {
        "Mod": "Insert Mod",
        etc...
     }
    "Select": {
         "Mod": "Select Mod",
         etc....
    }
```

File: src/app/item-popup/SocketDetailsSelectedPlug.tsx [hunk 2]
```diff
@@ -113,19 +152,12 @@ export default function SocketDetailsSelectedPlug({
...
-    } else if (DEFAULT_ORNAMENTS.includes(initialPlugHash)) {
-      insertName = canDoAWA ? t('Sockets.InsertOrnamentButton') : t('Sockets.SelectOrnamentButton');
-    } else {
-      insertName = canDoAWA ? t('Sockets.InsertModButton') : t('Sockets.SelectModButton');
-    }
-  } else {
-    insertName = canDoAWA ? t('Sockets.InsertModButton') : t('Sockets.SelectModButton');
-  }
+  // t(`Sockets.Insert`, { contextList: 'sockets' })
+  // t(`Sockets.Select`, { contextList: 'sockets' })
```
Comment (delphiactual) [inline src/app/item-popup/SocketDetailsSelectedPlug.tsx:156]: remove

File: src/locale/dim.json [hunk 3]
```diff
@@ -840,13 +840,23 @@
...
-    "SelectOrnamentButton": "Select Ornament",
-    "SelectShaderButton": "Select Shader",
+    "SelectAbility": "Select Ability",
+    "SelectAspect": "Select Aspect",
+    "SelectFragment": "Select Fragment",
+    "SelectMod": "Select Mod",
+    "SelectOrnament": "Select Ornament",
+    "SelectProjection": "Select Ghost Projection",
+    "SelectShader": "Select Shader",
+    "SelectTransmat": "Select Transmat Effect",
```
Comment (delphiactual) [inline src/locale/dim.json:859]: same as config/i18n.json

**Review by delphiactual — CHANGES_REQUESTED** [review]

**Comment by robojumper** [comment]
Thanks @delphiactual, I think I've addressed these.

**Review by delphiactual — APPROVED** [review]

**Pull request merged**